packages/create-react-app/createReactApp.js

```diff
@@ -182,27 +181,31 @@ createApp(
 );
 
 function createApp(name, verbose, version, useNpm, template) {
-  const root = path.resolve(name);
-  const appName = path.basename(root);
+  const appName = `portal-${name}-frontend`;
```

Comment:
I suppose the reason for not exposing this as `REACT_APP_MICROAPP_ID` is to enforce naming convention. It has to be documented/mentioned in the implementing microapps docs

Reply:
? REACT_APP_MICROAPP_ID is the microapp id. appName is the repo name and also the project directory.